src/Microsoft.ML.Api/SchemaDefinition.cs

```diff
@@ -287,9 +287,9 @@ public void RemoveMetadata(string kind)
         /// </summary>
         public Column this[string columnName]
         {
-#pragma warning disable TLC_NoThis // Do not use 'this' keyword for member access
+#pragma warning disable MSML_NoThis // Do not use 'this' keyword for member access
```

Comment:
Is this still valid?  We removed our own rule here, right?

Reply:
Whoops, missed one...